en/GitBranching.md

```diff
@@ -0,0 +1,64 @@
+# Git Branching
+
+## Semantic Versioning
+
+QGC uses semantic version for the version numbers associated with its releases.
+Semantic versioning is a 3-component number in the format of `vX.Y.Z`, where:
+
```

Comment:
@DonLakeFlyer I have subedited this page. I had to infer a few things, so please check it carefully.

I think it is worth outlining what you expect for each level X, Y ,Z. It is implied but not stated that a patch is bug fixes but only very minor functionality. I would normally expect minor release for new functionality, and major release for a major change in usage paradigm OR incompatibility.